fcrepo-configs/src/main/resources/config/clustered-postgesql/postgres-cluster.json.j2

```diff
@@ -0,0 +1,44 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "clustering" : {
+        "clusterName" : "fcrepo-cluster",
+        "configuration" : "${fcrepo.jgroups.configuration:config/jgroups-fcrepo-tcp.xml}"
```

Comment:
I'm totally in over my head here, but should this line be defaulting to the above provided config instead of the old one?

Reply:
@whikloj you are right on the money.  We extracted this from an ansible recipe which currently copies the files up and didn't get the file location changed.  Also blew it on the file extension, which I've fixed as well.